src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -901,9 +917,16 @@ void doContainer(DocumentManagement documentManagement,
       containerContents = documentManagement.listNodes(containerNode.getID(),
           true);
     } catch (SOAPFaultException soapFaultException) {
-      log.log(Level.WARNING,
-          "Error retrieving container contents: " + opentextDocId,
-          soapFaultException);
+      if (isPermissionsFailure(soapFaultException)) {
+        log.log(Level.FINE,
+            soapFaultException.getFault().getFaultString() + " {0}",
+            opentextDocId.toString());
+        response.respondNotFound();
```

Comment:
This is a behavioral change, isn't it? We used to do what, just not return child docids, and now we return 404. The rationale is that we do not have permission to list the children, that we require See + See Contents for all nodes? SGTM.